airbyte-cdk/python/airbyte_cdk/destinations/vector_db_based/__init__.py

```diff
@@ -2,24 +2,28 @@
 # Copyright (c) 2021 Airbyte, Inc., all rights reserved.
 #
 
-from .batcher import Batcher
-from .config import CohereEmbeddingConfigModel, FakeEmbeddingConfigModel, OpenAIEmbeddingConfigModel, ProcessingConfigModel
+from .config import CohereEmbeddingConfigModel, FakeEmbeddingConfigModel, OpenAIEmbeddingConfigModel, FromFieldEmbeddingConfigModel, AzureOpenAIEmbeddingConfigModel, OpenAICompatibleEmbeddingConfigModel, ProcessingConfigModel
 from .document_processor import Chunk, DocumentProcessor
 from .embedder import CohereEmbedder, Embedder, FakeEmbedder, OpenAIEmbedder
 from .indexer import Indexer
 from .writer import Writer
 
 __all__ = [
-    "Batcher",
     "CohereEmbeddingConfigModel",
     "FakeEmbeddingConfigModel",
     "OpenAIEmbeddingConfigModel",
+    "OpenAICompatibleEmbeddingConfigModel",
+    "AzureOpenAIEmbeddingConfigModel",
+    "FromFieldEmbeddingConfigModel",
     "ProcessingConfigModel",
     "DocumentProcessor",
     "Chunk",
     "Embedder",
     "FakeEmbedder",
     "OpenAIEmbedder",
+    "OpenAICompatibleEmbedder",
+    "FromFieldEmbedder",
+    "AzureOpenAIEmbedder",
     "CohereEmbedder",
     "Indexer",
     "Writer",
```

Comment:
Small nit, but this is becoming hard to scan over time and I had to scan up and down to see that the new additions were not duplicates. Alpha sort?